### ISSUE TITLE
1.17pre2 : missing lang key for blocks/items

### ISSUE THREAD

**Issue by Aledark789** [anegostudios/VintageStory-Issues#1989]
**Game Version:** 1.17pre2
**Platform:** Windows 
**Modded:** No 
**SP/MP:** Singleplayer

### Description
missing lang key for blocks/items

'''
Lang key not found: Crude shield
1.7.2022 17:58:13.621 [VerboseDebug] Lang key not found: Aged wooden shield
Lang key not found: Leather reinforced wooden shield
1.7.2022 17:58:13.674 [VerboseDebug] Lang key not found: Ornate leather reinforced wooden shield
Lang key not found: {0} Ornate {1} shield
1.7.2022 17:58:13.773 [VerboseDebug] Lang key not found: Blackguard shield
1.7.2022 18:03:07.227 [VerboseDebug] Lang key not found: game:block-handbooktitle-ore-sylvite-halite
1.7.2022 18:03:07.227 [VerboseDebug] Lang key not found: game:block-handbooktext-ore-sylvite-halite
1.7.2022 18:03:12.435 [VerboseDebug] Lang key not found: game:block-handbooktitle-quern-andesite
1.7.2022 18:03:12.436 [VerboseDebug] Lang key not found: game:block-handbooktext-quern-andesite
Lang key not found: Lost {0:0.##} hp through {1}
1.7.2022 18:22:27.737 [VerboseDebug] Lang key not found: game:block-handbooktitle-rock-basalt
1.7.2022 18:22:27.737 [VerboseDebug] Lang key not found: game:block-handbooktext-rock-basalt
1.7.2022 17:53:07.751 [VerboseDebug] Lang key not found: Shift-click
1.7.2022 17:53:07.751 [VerboseDebug] Lang key not found: Ctrl-click
1.7.2022 17:56:35.820 [VerboseDebug] Lang key not found: worldattribute-playerlives-desc
1.7.2022 17:56:35.821 [VerboseDebug] Lang key not found: worldattribute-saplingGrowthRate-desc
1.7.2022 17:56:35.822 [VerboseDebug] Lang key not found: worldattribute-temporalStormSleeping-desc
1.7.2022 17:57:53.557 [VerboseDebug] Lang key not found: Actions
1.7.2022 17:58:11.264 [VerboseDebug] Lang key not found: entitlement-vssupporter
1.7.2022 17:58:13.162 [VerboseDebug] Lang key not found: game:block-bookshelf
1.7.2022 17:58:13.180 [VerboseDebug] Lang key not found: game:block-clutter
1.7.2022 17:58:13.190 [VerboseDebug] Lang key not found: game:clutter-bookshelves-bookstand-empty
1.7.2022 17:58:13.195 [VerboseDebug] Lang key not found: game:clutter-bookshelves-lecturn-aged-book-opened
1.7.2022 18:03:07.227 [VerboseDebug] Lang key not found: game:block-handbooktitle-ore-sylvite-halite
1.7.2022 18:03:07.227 [VerboseDebug] Lang key not found: game:block-handbooktext-ore-sylvite-halite
1.7.2022 18:03:12.435 [VerboseDebug] Lang key not found: game:block-handbooktitle-quern-andesite
1.7.2022 18:03:12.436 [VerboseDebug] Lang key not found: game:block-handbooktext-quern-andesite

'''
### Logs
```
1.7.2022 17:53:06 [Debug] Creating game window with window mode Fullscreen
1.7.2022 17:53:06.659 [VerboseDebug] Loadsounds, found 46 sounds
1.7.2022 17:53:07.751 [VerboseDebug] Lang key not found: Shift-click
1.7.2022 17:53:07.751 [VerboseDebug] Lang key not found: Ctrl-click
1.7.2022 17:53:08.905 [VerboseDebug] [VSCreativeMod.dll] Check for mod systems in mod C:\VS1-17pre2\Mods\VSCreativeMod.dll
1.7.2022 17:53:08.907 [VerboseDebug] [VSCreativeMod.dll] Loaded assembly C:\VS1-17pre2\Mods\VSCreativeMod.dll
1.7.2022 17:53:08.924 [VerboseDebug] [VSEssentials.dll] Check for mod systems in mod C:\VS1-17pre2\Mods\VSEssentials.dll
1.7.2022 17:53:08.925 [VerboseDebug] [VSEssentials.dll] Loaded assembly C:\VS1-17pre2\Mods\VSEssentials.dll
1.7.2022 17:53:08.927 [VerboseDebug] [VSSurvivalMod.dll] Check for mod systems in mod C:\VS1-17pre2\Mods\VSSurvivalMod.dll
1.7.2022 17:53:08.929 [VerboseDebug] [VSSurvivalMod.dll] Loaded assembly C:\VS1-17pre2\Mods\VSSurvivalMod.dll
1.7.2022 17:53:09.094 [VerboseDebug] Left bottom main menu text is at 11,375/1034,0609375 w/h 262,5,38,9390625
1.7.2022 17:56:35.820 [VerboseDebug] Lang key not found: worldattribute-playerlives-desc
1.7.2022 17:56:35.821 [VerboseDebug] Lang key not found: worldattribute-saplingGrowthRate-desc
1.7.2022 17:56:35.822 [VerboseDebug] Lang key not found: worldattribute-temporalStormSleeping-desc
1.7.2022 17:57:53.557 [VerboseDebug] Lang key not found: Actions
1.7.2022 17:57:58.676 [VerboseDebug] Received server assets packet; waiting on light levels packet before decoding.
1.7.2022 17:57:58.680 [VerboseDebug] Received light levels
1.7.2022 17:57:58.682 [VerboseDebug] Received server assets
1.7.2022 17:57:58.770 [VerboseDebug] [VSCreativeMod.dll] Check for mod systems in mod C:\VS1-17pre2\Mods\VSCreativeMod.dll
1.7.2022 17:57:58.770 [VerboseDebug] [VSCreativeMod.dll] Loaded assembly C:\VS1-17pre2\Mods\VSCreativeMod.dll
1.7.2022 17:57:58.772 [VerboseDebug] [VSEssentials.dll] Check for mod systems in mod C:\VS1-17pre2\Mods\VSEssentials.dll
1.7.2022 17:57:58.772 [VerboseDebug] [VSEssentials.dll] Loaded assembly C:\VS1-17pre2\Mods\VSEssentials.dll
1.7.2022 17:57:58.774 [VerboseDebug] [VSSurvivalMod.dll] Check for mod systems in mod C:\VS1-17pre2\Mods\VSSurvivalMod.dll
1.7.2022 17:57:58.774 [VerboseDebug] [VSSurvivalMod.dll] Loaded assembly C:\VS1-17pre2\Mods\VSSurvivalMod.dll
1.7.2022 17:57:58.798 [VerboseDebug] 3 assemblies loaded
1.7.2022 17:57:58.799 [VerboseDebug] Instantiate mod systems for game
1.7.2022 17:57:58.799 [VerboseDebug] Instantiate mod systems for creative
1.7.2022 17:57:58.799 [VerboseDebug] Instantiate mod systems for survival
1.7.2022 17:57:58.800 [VerboseDebug] Single player game - starting mods on the client-side
1.7.2022 17:57:58.813 [VerboseDebug] Server assets - done step 1, next steps off-thread
1.7.2022 17:57:58.813 [VerboseDebug] Searching file system (including mods) for asset files
1.7.2022 17:57:58.861 [VerboseDebug] Populated 182 EntityTypes from server
1.7.2022 17:57:58.981 [VerboseDebug] Populated 2408 ItemTypes from server, highest ItemID is 2407
1.7.2022 17:57:59.133 [VerboseDebug] Finished the search for asset files
1.7.2022 17:57:59.181 [VerboseDebug] Loaded lang file: en
1.7.2022 17:57:59.203 [VerboseDebug] Patch file game:patches/fallingblocks.json, patch 0: Unmet IsValue condition (sandgravelsoil!=sandgravel)
1.7.2022 17:57:59.203 [VerboseDebug] Patch file game:patches/fallingblocks.json, patch 1: Unmet IsValue condition (sandgravelsoil!=sandgravel)
1.7.2022 17:57:59.203 [VerboseDebug] Patch file game:patches/fallingblocks.json, patch 2: Unmet IsValue condition (sandgravelsoil!=sandgravel)
1.7.2022 17:57:59.203 [VerboseDebug] Patch file game:patches/fallingblocks.json, patch 3: Unmet IsValue condition (sandgravelsoil!=sandgravel)
1.7.2022 17:57:59.203 [VerboseDebug] Patch file game:patches/fallingblocks.json, patch 4: Unmet IsValue condition (sandgravelsoil!=sandgravel)
1.7.2022 17:57:59.203 [VerboseDebug] Patch file game:patches/fallingblocks.json, patch 5: Unmet IsValue condition (sandgravelsoil!=sandgravel)
1.7.2022 17:57:59.203 [VerboseDebug] Patch file game:patches/fallingblocks.json, patch 6: Unmet IsValue condition (sandgravelsoil!=sandgravel)
1.7.2022 17:57:59.203 [VerboseDebug] Patch file game:patches/fallingblocks.json, patch 7: Unmet IsValue condition (sandgravelsoil!=sandgravel)
1.7.2022 17:57:59.203 [VerboseDebug] Patch file game:patches/fallingblocks.json, patch 8: Unmet IsValue condition (sandgravelsoil!=sandgravel)
1.7.2022 17:57:59.203 [VerboseDebug] Patch file game:patches/fallingblocks.json, patch 9: Unmet IsValue condition (sandgravelsoil!=sandgravel)
1.7.2022 17:57:59.204 [VerboseDebug] Patch file game:patches/keepinventory.json, patch 0: Unmet IsValue condition (keep!=drop)
1.7.2022 17:57:59.204 [VerboseDebug] Patchloader finished
1.7.2022 17:57:59.221 [VerboseDebug] All client-side assets loaded and patched; configs, shapes, textures and sounds are now available for loading
1.7.2022 17:57:59.228 [VerboseDebug] Starting to load entity shapes
1.7.2022 17:57:59.228 [VerboseDebug] Starting to load item shapes
1.7.2022 17:57:59.229 [VerboseDebug] Entity shape loading starting ...
1.7.2022 17:57:59.247 [VerboseDebug] [LoadShapes] Searched through items...
1.7.2022 17:57:59.931 [VerboseDebug] Populated 8470 BlockTypes from server, highest BlockID is 8469
1.7.2022 17:57:59.931 [VerboseDebug] Starting to load block shapes
1.7.2022 17:57:59.967 [VerboseDebug] Resolved blocks and items stacks and drops
1.7.2022 17:57:59.973 [VerboseDebug] [LoadShapes] Searching through blocks...
1.7.2022 17:58:00.029 [VerboseDebug] Entity shape loading completed
1.7.2022 17:58:00.059 [VerboseDebug] Loaded 23 ColorMap textures
1.7.2022 17:58:00.060 [VerboseDebug] Loadsounds, found 373 sounds
1.7.2022 17:58:00.172 [VerboseDebug] Server assets - composing entity texture atlas on main thread
1.7.2022 17:58:00.218 [VerboseDebug] Server assets - done step 5A
1.7.2022 17:58:00.220 [VerboseDebug] Server assets - done step 5B
1.7.2022 17:58:00.222 [VerboseDebug] Server assets - done entity texture atlas
1.7.2022 17:58:00.235 [VerboseDebug] [LoadShapes] Searched through 8470 blocks
1.7.2022 17:58:00.235 [VerboseDebug] [LoadShapes] Starting to parse block shapes...
1.7.2022 17:58:00.366 [VerboseDebug] [LoadShapes] parsed 520 shapes from JSON for items
1.7.2022 17:58:00.385 [VerboseDebug] [LoadShapes] loaded 2 block shapes in obj and gltf formats
1.7.2022 17:58:01.022 [VerboseDebug] Server assets - composing item texture atlas on main thread
1.7.2022 17:58:01.056 [VerboseDebug] [LoadShapes] parsed 527 shapes from JSON offthread for blocks
1.7.2022 17:58:01.056 [VerboseDebug] [LoadShapes] parsed 527 shapes from JSON for 8470 blocks, some others done offthread
1.7.2022 17:58:01.059 [VerboseDebug] BlockTextureAtlas start collecting textures (already holds 23 colormap textures)
1.7.2022 17:58:01.061 [VerboseDebug] Server assets - done step 6A
1.7.2022 17:58:01.068 [VerboseDebug] Server assets - done item texture atlas
1.7.2022 17:58:01.185 [VerboseDebug] Server assets - done step 6B
1.7.2022 17:58:01.674 [VerboseDebug] Server assets - done item tesselation spread over 23 frames
1.7.2022 17:58:02.302 [VerboseDebug] Starting to load sounds fully
1.7.2022 17:58:02.303 [VerboseDebug] Server assets - composing block texture atlas on main thread
1.7.2022 17:58:02.305 [VerboseDebug] Load sound asset sounds/environment/wind.ogg
1.7.2022 17:58:02.338 [VerboseDebug] Server assets - done step 8A
1.7.2022 17:58:02.357 [VerboseDebug] Server assets - done block texture atlas
1.7.2022 17:58:02.364 [VerboseDebug] Load sound asset sounds/effect/rift.ogg
1.7.2022 17:58:02.441 [VerboseDebug] Load sound asset sounds/effect/tempstab-drain.ogg
1.7.2022 17:58:02.445 [VerboseDebug] Server assets - done step 8B
1.7.2022 17:58:02.452 [VerboseDebug] Server assets - done step 9
1.7.2022 17:58:02.586 [VerboseDebug] Server assets - done texture atlas frame buffer
1.7.2022 17:58:02.736 [VerboseDebug] Load sound asset sounds/effect/tempstab-low.ogg
1.7.2022 17:58:02.997 [VerboseDebug] Load sound asset sounds/effect/tempstab-verylow.ogg
1.7.2022 17:58:03.188 [VerboseDebug] Load sound asset sounds/environment/creek.ogg
1.7.2022 17:58:03.358 [VerboseDebug] Load sound asset sounds/environment/drip1.ogg
1.7.2022 17:58:03.391 [VerboseDebug] Load sound asset sounds/environment/drip2.ogg
1.7.2022 17:58:03.420 [VerboseDebug] Load sound asset sounds/environment/drip3.ogg
1.7.2022 17:58:03.450 [VerboseDebug] Load sound asset sounds/environment/fire.ogg
1.7.2022 17:58:03.492 [VerboseDebug] Load sound asset sounds/environment/fireplace.ogg
1.7.2022 17:58:03.595 [VerboseDebug] Load sound asset sounds/environment/largesplash1.ogg
1.7.2022 17:58:03.626 [VerboseDebug] Load sound asset sounds/environment/largesplash2.ogg
1.7.2022 17:58:03.658 [VerboseDebug] Load sound asset sounds/environment/lava.ogg
1.7.2022 17:58:03.828 [VerboseDebug] Load sound asset sounds/environment/mediumsplash.ogg
1.7.2022 17:58:03.859 [VerboseDebug] Load sound asset sounds/environment/smallsplash.ogg
1.7.2022 17:58:03.889 [VerboseDebug] Load sound asset sounds/environment/underwater.ogg
1.7.2022 17:58:04.007 [VerboseDebug] Load sound asset sounds/environment/waterfall.ogg
1.7.2022 17:58:04.030 [VerboseDebug] Server assets - done block tesselation
1.7.2022 17:58:04.139 [VerboseDebug] Load sound asset sounds/environment/waterwaves.ogg
1.7.2022 17:58:04.267 [VerboseDebug] Load sound asset sounds/weather/tracks/hail.ogg
1.7.2022 17:58:04.298 [VerboseDebug] Load sound asset sounds/weather/tracks/lowtremble.ogg
1.7.2022 17:58:04.344 [VerboseDebug] Load sound asset sounds/weather/tracks/rain-leafless.ogg
1.7.2022 17:58:04.408 [VerboseDebug] Load sound asset sounds/weather/tracks/rain-leafy.ogg
1.7.2022 17:58:04.484 [VerboseDebug] Load sound asset sounds/weather/tracks/verylowtremble.ogg
1.7.2022 17:58:04.546 [VerboseDebug] Load sound asset sounds/weather/lightning-distant.ogg
1.7.2022 17:58:04.625 [VerboseDebug] Load sound asset sounds/weather/lightning-near.ogg
1.7.2022 17:58:04.686 [VerboseDebug] Load sound asset sounds/weather/lightning-nodistance.ogg
1.7.2022 17:58:04.717 [VerboseDebug] Load sound asset sounds/weather/lightning-verynear.ogg
1.7.2022 17:58:04.809 [VerboseDebug] Load sound asset sounds/weather/lowgrumble.ogg
1.7.2022 17:58:04.870 [VerboseDebug] Load sound asset sounds/weather/wind-leafless.ogg
1.7.2022 17:58:04.947 [VerboseDebug] Load sound asset sounds/weather/wind-leafy.ogg
1.7.2022 17:58:05.010 [VerboseDebug] Loaded highest priority sound assets
1.7.2022 17:58:10.875 [VerboseDebug] --- Server started ---
1.7.2022 17:58:11.264 [VerboseDebug] Lang key not found: entitlement-vssupporter
1.7.2022 17:58:11.377 [VerboseDebug] Handling ServerIdentification packet
1.7.2022 17:58:11.382 [VerboseDebug] Starting system: Core
1.7.2022 17:58:11.383 [VerboseDebug] Starting system: EntityPartitioning
1.7.2022 17:58:11.384 [VerboseDebug] Starting system: ErrorReporter
1.7.2022 17:58:11.385 [VerboseDebug] Starting system: SaplingControl
1.7.2022 17:58:11.385 [VerboseDebug] Starting system: Core
1.7.2022 17:58:11.385 [VerboseDebug] Starting system: SurvivalCoreSystem
1.7.2022 17:58:11.394 [VerboseDebug] Starting system: ModCompatiblityUtil
1.7.2022 17:58:11.394 [VerboseDebug] Starting system: ModJsonPatchLoader
1.7.2022 17:58:11.394 [VerboseDebug] Starting system: FallingBlockParticlesModSystem
1.7.2022 17:58:11.394 [VerboseDebug] Starting system: CharacterExtraDialogs
1.7.2022 17:58:11.396 [VerboseDebug] Starting system: EntityNameTagRendererRegistry
1.7.2022 17:58:11.396 [VerboseDebug] Starting system: POIRegistry
1.7.2022 17:58:11.396 [VerboseDebug] Starting system: RoomRegistry
1.7.2022 17:58:11.397 [VerboseDebug] Starting system: WeatherSystemCommands
1.7.2022 17:58:11.397 [VerboseDebug] Starting system: WeatherSystemClient
1.7.2022 17:58:11.416 [VerboseDebug] Starting system: WorldMapManager
1.7.2022 17:58:11.419 [VerboseDebug] Starting system: WorldEdit
1.7.2022 17:58:11.423 [VerboseDebug] Starting system: GenFromHeightmap
1.7.2022 17:58:11.423 [VerboseDebug] Starting system: GenMaps
1.7.2022 17:58:11.424 [VerboseDebug] Starting system: DebugSystem
1.7.2022 17:58:11.424 [VerboseDebug] Starting system: UpgradeTasks
1.7.2022 17:58:11.424 [VerboseDebug] Starting system: MyceliumSystem
1.7.2022 17:58:11.424 [VerboseDebug] Starting system: ModSystemAuction
1.7.2022 17:58:11.427 [VerboseDebug] Starting system: ModSystemHandbook
1.7.2022 17:58:11.429 [VerboseDebug] Starting system: LiquidItemStackRenderer
1.7.2022 17:58:11.429 [VerboseDebug] Starting system: CharacterSystem
1.7.2022 17:58:11.432 [VerboseDebug] Starting system: FruitingSystem
1.7.2022 17:58:11.432 [VerboseDebug] Starting system: ModSystemRifts
1.7.2022 17:58:11.439 [VerboseDebug] Starting system: ModSystemRiftWeather
1.7.2022 17:58:11.440 [VerboseDebug] Starting system: ModTemperature
1.7.2022 17:58:11.440 [VerboseDebug] Starting system: SystemTemporalStability
1.7.2022 17:58:11.441 [VerboseDebug] Starting system: MealMeshCache
1.7.2022 17:58:11.441 [VerboseDebug] Starting system: ModSystemWearableStats
1.7.2022 17:58:11.442 [VerboseDebug] Starting system: ModSystemBlockReinforcement
1.7.2022 17:58:11.445 [VerboseDebug] Starting system: ChiselBlockModelCache
1.7.2022 17:58:11.445 [VerboseDebug] Starting system: TemporalStabilityEffects
1.7.2022 17:58:11.450 [VerboseDebug] Starting system: ModJournal
1.7.2022 17:58:11.453 [VerboseDebug] Starting system: ModLootRandomizer
1.7.2022 17:58:11.453 [VerboseDebug] Starting system: ModSleeping
1.7.2022 17:58:11.459 [VerboseDebug] Starting system: TeleporterManager
1.7.2022 17:58:11.461 [VerboseDebug] Starting system: MechanicalPowerMod
1.7.2022 17:58:11.462 [VerboseDebug] Starting system: TradeHandbookInfo
1.7.2022 17:58:11.462 [VerboseDebug] Starting system: LoadColorMaps
1.7.2022 17:58:11.462 [VerboseDebug] Starting system: GenStructures
1.7.2022 17:58:11.462 [VerboseDebug] Starting system: ClothManager
1.7.2022 17:58:11.466 [VerboseDebug] Starting system: GenStructuresPosPass
1.7.2022 17:58:11.466 [VerboseDebug] Starting system: RecipeRegistrySystem
1.7.2022 17:58:11.466 [VerboseDebug] Starting system: TraderOutfits
1.7.2022 17:58:11.644 [VerboseDebug] Sending request to join server
1.7.2022 17:58:11.718 [VerboseDebug] World configs received and block tesselation complete
1.7.2022 17:58:11.734 [VerboseDebug] Trigger mod event OnBlockTexturesLoaded
1.7.2022 17:58:11.776 [VerboseDebug] Slow to load clientSystem modhandler
1.7.2022 17:58:11.821 [VerboseDebug] Slow to load clientSystem gdm
1.7.2022 17:58:11.855 [VerboseDebug] Slow to load clientSystem ret
1.7.2022 17:58:11.940 [VerboseDebug] Done client systems OnLoaded
1.7.2022 17:58:11.971 [VerboseDebug] Slow to load blocks game:oillamp-east,game:oillamp-south,game:oillamp-west,game:clayoven-north
1.7.2022 17:58:12.019 [VerboseDebug] Slow to load blocks game:condenser-north,game:condenser-east,game:condenser-south,game:condenser-west
1.7.2022 17:58:12.187 [VerboseDebug] Slow to load blocks game:fruittree-stem,game:fruittree-branch,game:fruittree-cutting,game:fruittree-foliage
1.7.2022 17:58:12.269 [VerboseDebug] Slow to load blocks game:altar-east,game:altar-south,game:altar-west,game:forge
1.7.2022 17:58:12.321 [VerboseDebug] Slow to load blocks game:door-plank-west-up-opened-right,game:door-log-west-up-opened-right,game:echochamber-north,game:echochamber-east
1.7.2022 17:58:12.341 [VerboseDebug] Slow to load blocks game:pan-wooden,game:woodenpath-ns,game:woodenpath-we,game:pitkiln
1.7.2022 17:58:12.355 [VerboseDebug] Done blocks OnLoaded
1.7.2022 17:58:12.722 [VerboseDebug] Done items OnLoaded
1.7.2022 17:58:12.736 [VerboseDebug] All clientside asset loading complete, game launch can proceed
1.7.2022 17:58:12.745 [VerboseDebug] Loading sound file may stutter game:sounds/voice/saxophone.ogg
1.7.2022 17:58:12.814 [VerboseDebug] Received player data for playeruid p0z1ChFD8hsSr35OZf0pCLyG
1.7.2022 17:58:12.817 [VerboseDebug] Received inventory contents hotbar-p0z1ChFD8hsSr35OZf0pCLyG
1.7.2022 17:58:12.817 [VerboseDebug] Received inventory contents creative-p0z1ChFD8hsSr35OZf0pCLyG
1.7.2022 17:58:12.877 [VerboseDebug] Received inventory contents backpack-p0z1ChFD8hsSr35OZf0pCLyG
1.7.2022 17:58:12.877 [VerboseDebug] Received inventory contents ground-p0z1ChFD8hsSr35OZf0pCLyG
1.7.2022 17:58:12.877 [VerboseDebug] Received inventory contents mouse-p0z1ChFD8hsSr35OZf0pCLyG
1.7.2022 17:58:12.878 [VerboseDebug] Received inventory contents craftinggrid-p0z1ChFD8hsSr35OZf0pCLyG
1.7.2022 17:58:12.889 [VerboseDebug] Received inventory contents character-p0z1ChFD8hsSr35OZf0pCLyG
1.7.2022 17:58:12.889 [VerboseDebug] Informing clientsystems playerdata received
1.7.2022 17:58:12.907 [VerboseDebug] GuiDialogInventory: starting composeGUI
1.7.2022 17:58:12.931 [VerboseDebug] GuiDialogInventory: done composeGUI
1.7.2022 17:58:13.162 [VerboseDebug] Lang key not found: game:block-bookshelf
1.7.2022 17:58:13.180 [VerboseDebug] Lang key not found: game:block-clutter
1.7.2022 17:58:13.190 [VerboseDebug] Lang key not found: game:clutter-bookshelves-bookstand-empty
1.7.2022 17:58:13.195 [VerboseDebug] Lang key not found: game:clutter-bookshelves-lecturn-aged-book-opened
1.7.2022 17:58:13.382 [VerboseDebug] Done handling playerdata packet
1.7.2022 17:58:13.387 [VerboseDebug] Received level finalize
1.7.2022 17:58:13.393 [VerboseDebug] Done level finalize clientsystems
1.7.2022 17:58:13.433 [VerboseDebug] Done init WeatherSystemClient
1.7.2022 17:58:13.575 [VerboseDebug] Starting initialising handbook
1.7.2022 17:58:13.616 [VerboseDebug] Lang key not found: Crude shield
1.7.2022 17:58:13.621 [VerboseDebug] Lang key not found: Aged wooden shield
1.7.2022 17:58:13.621 [VerboseDebug] Lang key not found: {0} shield
1.7.2022 17:58:13.668 [VerboseDebug] Done all stacks for handbook
1.7.2022 17:58:13.674 [VerboseDebug] Lang key not found: Leather reinforced wooden shield
1.7.2022 17:58:13.674 [VerboseDebug] Lang key not found: Ornate leather reinforced wooden shield
1.7.2022 17:58:13.712 [VerboseDebug] Done creating handbook index GUI
1.7.2022 17:58:13.713 [VerboseDebug] Done initialising handbook
1.7.2022 17:58:13.718 [VerboseDebug] Done scanning for liquid containers 
1.7.2022 17:58:13.718 [VerboseDebug] Done wearable stats
1.7.2022 17:58:13.722 [VerboseDebug] Done init huge gears
1.7.2022 17:58:13.727 [VerboseDebug] Lang key not found: {0} Ornate {1} shield
1.7.2022 17:58:13.773 [VerboseDebug] Lang key not found: Blackguard shield
1.7.2022 17:58:13.773 [VerboseDebug] Done traders handbook stuff
1.7.2022 17:58:13.776 [VerboseDebug] Done level finalize
1.7.2022 17:59:06.546 [VerboseDebug] Loading sound file may stutter game:sounds/voice/sax.ogg
1.7.2022 17:59:12.193 [VerboseDebug] Loading sound file may stutter game:sounds/voice/tuba.ogg
1.7.2022 17:59:12.767 [VerboseDebug] Loading sound file may stutter game:sounds/voice/trumpet.ogg
1.7.2022 17:59:15.076 [VerboseDebug] Loading sound file may stutter game:sounds/voice/oboe.ogg
1.7.2022 17:59:15.699 [VerboseDebug] Loading sound file may stutter game:sounds/voice/clarinet.ogg
1.7.2022 17:59:17.627 [VerboseDebug] Loading sound file may stutter game:sounds/voice/altoflute.ogg
1.7.2022 17:59:22.761 [VerboseDebug] Loading sound file may stutter game:sounds/voice/harmonica.ogg
1.7.2022 18:00:02.346 [VerboseDebug] Received player data for playeruid p0z1ChFD8hsSr35OZf0pCLyG
1.7.2022 18:00:02.346 [VerboseDebug] Received inventory contents hotbar-p0z1ChFD8hsSr35OZf0pCLyG
1.7.2022 18:00:02.346 [VerboseDebug] Received inventory contents creative-p0z1ChFD8hsSr35OZf0pCLyG
1.7.2022 18:00:02.346 [VerboseDebug] Received inventory contents backpack-p0z1ChFD8hsSr35OZf0pCLyG
1.7.2022 18:00:02.346 [VerboseDebug] Received inventory contents ground-p0z1ChFD8hsSr35OZf0pCLyG
1.7.2022 18:00:02.346 [VerboseDebug] Received inventory contents mouse-p0z1ChFD8hsSr35OZf0pCLyG
1.7.2022 18:00:02.346 [VerboseDebug] Received inventory contents craftinggrid-p0z1ChFD8hsSr35OZf0pCLyG
1.7.2022 18:00:02.349 [VerboseDebug] Received inventory contents character-p0z1ChFD8hsSr35OZf0pCLyG
1.7.2022 18:00:02.350 [VerboseDebug] Done handling playerdata packet
1.7.2022 18:00:02.350 [VerboseDebug] Received player data for playeruid p0z1ChFD8hsSr35OZf0pCLyG
1.7.2022 18:00:02.350 [VerboseDebug] Done handling playerdata packet
1.7.2022 18:00:19.046 [VerboseDebug] Left bottom main menu text is at 11,375/1034,0609375 w/h 262,5,38,9390625
1.7.2022 18:00:19.109 [VerboseDebug] Main menu: memory before/after GC : 1107146328
1.7.2022 18:01:05.135 [VerboseDebug] Received server assets packet; waiting on light levels packet before decoding.
1.7.2022 18:01:05.135 [VerboseDebug] Received light levels
1.7.2022 18:01:05.136 [VerboseDebug] Received server assets
1.7.2022 18:01:05.250 [VerboseDebug] [VSCreativeMod.dll] Check for mod systems in mod C:\VS1-17pre2\Mods\VSCreativeMod.dll
1.7.2022 18:01:05.250 [VerboseDebug] [VSCreativeMod.dll] Loaded assembly C:\VS1-17pre2\Mods\VSCreativeMod.dll
1.7.2022 18:01:05.252 [VerboseDebug] [VSEssentials.dll] Check for mod systems in mod C:\VS1-17pre2\Mods\VSEssentials.dll
1.7.2022 18:01:05.252 [VerboseDebug] [VSEssentials.dll] Loaded assembly C:\VS1-17pre2\Mods\VSEssentials.dll
1.7.2022 18:01:05.254 [VerboseDebug] [VSSurvivalMod.dll] Check for mod systems in mod C:\VS1-17pre2\Mods\VSSurvivalMod.dll
1.7.2022 18:01:05.254 [VerboseDebug] [VSSurvivalMod.dll] Loaded assembly C:\VS1-17pre2\Mods\VSSurvivalMod.dll
1.7.2022 18:01:05.266 [VerboseDebug] 3 assemblies loaded
1.7.2022 18:01:05.266 [VerboseDebug] Instantiate mod systems for game
1.7.2022 18:01:05.267 [VerboseDebug] Instantiate mod systems for creative
1.7.2022 18:01:05.267 [VerboseDebug] Instantiate mod systems for survival
1.7.2022 18:01:05.267 [VerboseDebug] Single player game - starting mods on the client-side
1.7.2022 18:01:05.268 [VerboseDebug] Server assets - done step 1, next steps off-thread
1.7.2022 18:01:05.268 [VerboseDebug] Searching file system (including mods) for asset files
1.7.2022 18:01:05.297 [VerboseDebug] Populated 182 EntityTypes from server
1.7.2022 18:01:05.427 [VerboseDebug] Populated 2408 ItemTypes from server, highest ItemID is 2407
1.7.2022 18:01:05.840 [VerboseDebug] Finished the search for asset files
1.7.2022 18:01:05.900 [VerboseDebug] Loaded lang file: en
1.7.2022 18:01:05.933 [VerboseDebug] Patch file game:patches/fallingblocks.json, patch 0: Unmet IsValue condition (sandgravelsoil!=sandgravel)
1.7.2022 18:01:05.933 [VerboseDebug] Patch file game:patches/fallingblocks.json, patch 1: Unmet IsValue condition (sandgravelsoil!=sandgravel)
1.7.2022 18:01:05.933 [VerboseDebug] Patch file game:patches/fallingblocks.json, patch 2: Unmet IsValue condition (sandgravelsoil!=sandgravel)
1.7.2022 18:01:05.933 [VerboseDebug] Patch file game:patches/fallingblocks.json, patch 3: Unmet IsValue condition (sandgravelsoil!=sandgravel)
1.7.2022 18:01:05.933 [VerboseDebug] Patch file game:patches/fallingblocks.json, patch 4: Unmet IsValue condition (sandgravelsoil!=sandgravel)
1.7.2022 18:01:05.933 [VerboseDebug] Patch file game:patches/fallingblocks.json, patch 5: Unmet IsValue condition (sandgravelsoil!=sandgravel)
1.7.2022 18:01:05.933 [VerboseDebug] Patch file game:patches/fallingblocks.json, patch 6: Unmet IsValue condition (sandgravelsoil!=sandgravel)
1.7.2022 18:01:05.933 [VerboseDebug] Patch file game:patches/fallingblocks.json, patch 7: Unmet IsValue condition (sandgravelsoil!=sandgravel)
1.7.2022 18:01:05.933 [VerboseDebug] Patch file game:patches/fallingblocks.json, patch 8: Unmet IsValue condition (sandgravelsoil!=sandgravel)
1.7.2022 18:01:05.933 [VerboseDebug] Patch file game:patches/fallingblocks.json, patch 9: Unmet IsValue condition (sandgravelsoil!=sandgravel)
1.7.2022 18:01:05.934 [VerboseDebug] Patch file game:patches/keepinventory.json, patch 0: Unmet IsValue condition (keep!=drop)
1.7.2022 18:01:05.943 [VerboseDebug] Patchloader finished
1.7.2022 18:01:05.944 [VerboseDebug] All client-side assets loaded and patched; configs, shapes, textures and sounds are now available for loading
1.7.2022 18:01:05.944 [VerboseDebug] Starting to load entity shapes
1.7.2022 18:01:05.945 [VerboseDebug] Entity shape loading starting ...
1.7.2022 18:01:05.946 [VerboseDebug] Starting to load item shapes
1.7.2022 18:01:05.948 [VerboseDebug] [LoadShapes] Searched through items...
1.7.2022 18:01:06.165 [VerboseDebug] Populated 8470 BlockTypes from server, highest BlockID is 8469
1.7.2022 18:01:06.165 [VerboseDebug] Starting to load block shapes
1.7.2022 18:01:06.179 [VerboseDebug] Resolved blocks and items stacks and drops
1.7.2022 18:01:06.179 [VerboseDebug] [LoadShapes] Searching through blocks...
1.7.2022 18:01:06.392 [VerboseDebug] Entity shape loading completed
1.7.2022 18:01:06.407 [VerboseDebug] Loaded 23 ColorMap textures
1.7.2022 18:01:06.408 [VerboseDebug] Loadsounds, found 373 sounds
1.7.2022 18:01:06.474 [VerboseDebug] [LoadShapes] Searched through 8470 blocks
1.7.2022 18:01:06.474 [VerboseDebug] [LoadShapes] Starting to parse block shapes...
1.7.2022 18:01:06.474 [VerboseDebug] [LoadShapes] loaded 2 block shapes in obj and gltf formats
1.7.2022 18:01:06.500 [VerboseDebug] Server assets - composing entity texture atlas on main thread
1.7.2022 18:01:06.542 [VerboseDebug] Server assets - done step 5A
1.7.2022 18:01:06.543 [VerboseDebug] Server assets - done entity texture atlas
1.7.2022 18:01:06.651 [VerboseDebug] Server assets - done step 5B
1.7.2022 18:01:07.040 [VerboseDebug] [LoadShapes] parsed 512 shapes from JSON offthread for blocks
1.7.2022 18:01:07.040 [VerboseDebug] [LoadShapes] parsed 542 shapes from JSON for 8470 blocks, some others done offthread
1.7.2022 18:01:07.070 [VerboseDebug] [LoadShapes] parsed 520 shapes from JSON for items
1.7.2022 18:01:07.085 [VerboseDebug] BlockTextureAtlas start collecting textures (already holds 23 colormap textures)
1.7.2022 18:01:07.364 [VerboseDebug] Server assets - composing item texture atlas on main thread
1.7.2022 18:01:07.407 [VerboseDebug] Server assets - done step 6A
1.7.2022 18:01:07.414 [VerboseDebug] Server assets - done item texture atlas
1.7.2022 18:01:07.521 [VerboseDebug] Server assets - done step 6B
1.7.2022 18:01:08.896 [VerboseDebug] Starting to load sounds fully
1.7.2022 18:01:08.896 [VerboseDebug] Server assets - composing block texture atlas on main thread
1.7.2022 18:01:08.896 [VerboseDebug] Load sound asset sounds/environment/wind.ogg
1.7.2022 18:01:08.935 [VerboseDebug] Server assets - done step 8A
1.7.2022 18:01:08.938 [VerboseDebug] Load sound asset sounds/effect/rift.ogg
1.7.2022 18:01:08.953 [VerboseDebug] Server assets - done block texture atlas
1.7.2022 18:01:09.014 [VerboseDebug] Load sound asset sounds/effect/tempstab-drain.ogg
1.7.2022 18:01:09.058 [VerboseDebug] Server assets - done step 8B
1.7.2022 18:01:09.060 [VerboseDebug] Server assets - done step 9
1.7.2022 18:01:09.229 [VerboseDebug] Server assets - done texture atlas frame buffer
1.7.2022 18:01:09.244 [VerboseDebug] Load sound asset sounds/effect/tempstab-low.ogg
1.7.2022 18:01:09.599 [VerboseDebug] Load sound asset sounds/effect/tempstab-verylow.ogg
1.7.2022 18:01:09.824 [VerboseDebug] Load sound asset sounds/environment/creek.ogg
1.7.2022 18:01:09.962 [VerboseDebug] Load sound asset sounds/environment/drip1.ogg
1.7.2022 18:01:09.993 [VerboseDebug] Load sound asset sounds/environment/drip2.ogg
1.7.2022 18:01:10.024 [VerboseDebug] Load sound asset sounds/environment/drip3.ogg
1.7.2022 18:01:10.060 [VerboseDebug] Load sound asset sounds/environment/fire.ogg
1.7.2022 18:01:10.117 [VerboseDebug] Load sound asset sounds/environment/fireplace.ogg
1.7.2022 18:01:10.123 [VerboseDebug] Server assets - done item tesselation spread over 45 frames
1.7.2022 18:01:10.210 [VerboseDebug] Load sound asset sounds/environment/largesplash1.ogg
1.7.2022 18:01:10.241 [VerboseDebug] Load sound asset sounds/environment/largesplash2.ogg
1.7.2022 18:01:10.272 [VerboseDebug] Load sound asset sounds/environment/lava.ogg
1.7.2022 18:01:10.347 [VerboseDebug] Server assets - done block tesselation
1.7.2022 18:01:10.412 [VerboseDebug] Load sound asset sounds/environment/mediumsplash.ogg
1.7.2022 18:01:10.443 [VerboseDebug] Load sound asset sounds/environment/smallsplash.ogg
1.7.2022 18:01:10.475 [VerboseDebug] Load sound asset sounds/environment/underwater.ogg
1.7.2022 18:01:10.583 [VerboseDebug] Load sound asset sounds/environment/waterfall.ogg
1.7.2022 18:01:10.660 [VerboseDebug] Load sound asset sounds/environment/waterwaves.ogg
1.7.2022 18:01:10.799 [VerboseDebug] Load sound asset sounds/weather/tracks/hail.ogg
1.7.2022 18:01:10.830 [VerboseDebug] Load sound asset sounds/weather/tracks/lowtremble.ogg
1.7.2022 18:01:10.876 [VerboseDebug] Load sound asset sounds/weather/tracks/rain-leafless.ogg
1.7.2022 18:01:10.937 [VerboseDebug] Load sound asset sounds/weather/tracks/rain-leafy.ogg
1.7.2022 18:01:11.015 [VerboseDebug] Load sound asset sounds/weather/tracks/verylowtremble.ogg
1.7.2022 18:01:11.077 [VerboseDebug] Load sound asset sounds/weather/lightning-distant.ogg
1.7.2022 18:01:11.155 [VerboseDebug] Load sound asset sounds/weather/lightning-near.ogg
1.7.2022 18:01:11.219 [VerboseDebug] Load sound asset sounds/weather/lightning-nodistance.ogg
1.7.2022 18:01:11.267 [VerboseDebug] Load sound asset sounds/weather/lightning-verynear.ogg
1.7.2022 18:01:11.363 [VerboseDebug] Load sound asset sounds/weather/lowgrumble.ogg
1.7.2022 18:01:11.427 [VerboseDebug] Load sound asset sounds/weather/wind-leafless.ogg
1.7.2022 18:01:11.507 [VerboseDebug] Load sound asset sounds/weather/wind-leafy.ogg
1.7.2022 18:01:11.565 [VerboseDebug] Handling ServerIdentification packet
1.7.2022 18:01:11.565 [VerboseDebug] Starting system: Core
1.7.2022 18:01:11.565 [VerboseDebug] Starting system: EntityPartitioning
1.7.2022 18:01:11.565 [VerboseDebug] Starting system: ErrorReporter
1.7.2022 18:01:11.565 [VerboseDebug] Starting system: SaplingControl
1.7.2022 18:01:11.565 [VerboseDebug] Starting system: Core
1.7.2022 18:01:11.565 [VerboseDebug] Starting system: SurvivalCoreSystem
1.7.2022 18:01:11.571 [VerboseDebug] Loaded highest priority sound assets
1.7.2022 18:01:11.802 [VerboseDebug] Starting system: ModCompatiblityUtil
1.7.2022 18:01:11.802 [VerboseDebug] Starting system: ModJsonPatchLoader
1.7.2022 18:01:11.802 [VerboseDebug] Starting system: FallingBlockParticlesModSystem
1.7.2022 18:01:11.802 [VerboseDebug] Starting system: CharacterExtraDialogs
1.7.2022 18:01:11.802 [VerboseDebug] Starting system: EntityNameTagRendererRegistry
1.7.2022 18:01:11.802 [VerboseDebug] Starting system: POIRegistry
1.7.2022 18:01:11.802 [VerboseDebug] Starting system: RoomRegistry
1.7.2022 18:01:11.802 [VerboseDebug] Starting system: WeatherSystemCommands
1.7.2022 18:01:11.802 [VerboseDebug] Starting system: WeatherSystemClient
1.7.2022 18:01:11.897 [VerboseDebug] Starting system: WorldMapManager
1.7.2022 18:01:11.897 [VerboseDebug] Starting system: WorldEdit
1.7.2022 18:01:11.897 [VerboseDebug] Starting system: GenFromHeightmap
1.7.2022 18:01:11.897 [VerboseDebug] Starting system: GenMaps
1.7.2022 18:01:11.897 [VerboseDebug] Starting system: DebugSystem
1.7.2022 18:01:11.897 [VerboseDebug] Starting system: UpgradeTasks
1.7.2022 18:01:11.897 [VerboseDebug] Starting system: MyceliumSystem
1.7.2022 18:01:11.897 [VerboseDebug] Starting system: ModSystemAuction
1.7.2022 18:01:11.897 [VerboseDebug] Starting system: ModSystemHandbook
1.7.2022 18:01:11.897 [VerboseDebug] Starting system: LiquidItemStackRenderer
1.7.2022 18:01:11.897 [VerboseDebug] Starting system: CharacterSystem
1.7.2022 18:01:11.897 [VerboseDebug] Starting system: FruitingSystem
1.7.2022 18:01:11.897 [VerboseDebug] Starting system: ModSystemRifts
1.7.2022 18:01:11.949 [VerboseDebug] Starting system: ModSystemRiftWeather
1.7.2022 18:01:11.950 [VerboseDebug] Starting system: ModTemperature
1.7.2022 18:01:11.950 [VerboseDebug] Starting system: SystemTemporalStability
1.7.2022 18:01:11.950 [VerboseDebug] Starting system: MealMeshCache
1.7.2022 18:01:11.950 [VerboseDebug] Starting system: ModSystemWearableStats
1.7.2022 18:01:11.950 [VerboseDebug] Starting system: ModSystemBlockReinforcement
1.7.2022 18:01:11.950 [VerboseDebug] Starting system: ChiselBlockModelCache
1.7.2022 18:01:11.950 [VerboseDebug] Starting system: TemporalStabilityEffects
1.7.2022 18:01:11.974 [VerboseDebug] Starting system: ModJournal
1.7.2022 18:01:11.974 [VerboseDebug] Starting system: ModLootRandomizer
1.7.2022 18:01:11.974 [VerboseDebug] Starting system: ModSleeping
1.7.2022 18:01:11.979 [VerboseDebug] Starting system: TeleporterManager
1.7.2022 18:01:11.979 [VerboseDebug] Starting system: MechanicalPowerMod
1.7.2022 18:01:11.979 [VerboseDebug] Starting system: TradeHandbookInfo
1.7.2022 18:01:11.979 [VerboseDebug] Starting system: LoadColorMaps
1.7.2022 18:01:11.979 [VerboseDebug] Starting system: GenStructures
1.7.2022 18:01:11.979 [VerboseDebug] Starting system: ClothManager
1.7.2022 18:01:11.979 [VerboseDebug] Starting system: GenStructuresPosPass
1.7.2022 18:01:11.979 [VerboseDebug] Starting system: RecipeRegistrySystem
1.7.2022 18:01:11.979 [VerboseDebug] Starting system: TraderOutfits
1.7.2022 18:01:12.186 [VerboseDebug] Sending request to join server
1.7.2022 18:01:12.273 [VerboseDebug] World configs received and block tesselation complete
1.7.2022 18:01:12.278 [VerboseDebug] Trigger mod event OnBlockTexturesLoaded
1.7.2022 18:01:12.315 [VerboseDebug] Slow to load clientSystem modhandler
1.7.2022 18:01:12.347 [VerboseDebug] Done client systems OnLoaded
1.7.2022 18:01:12.431 [VerboseDebug] Slow to load blocks game:flower-catmint-free,game:flower-cornflower-free,game:flower-forgetmenot-free,game:flower-edelweiss-free
1.7.2022 18:01:12.581 [VerboseDebug] Slow to load blocks game:fruittree-stem,game:fruittree-branch,game:fruittree-cutting,game:fruittree-foliage
1.7.2022 18:01:12.635 [VerboseDebug] Slow to load blocks game:roughhewnfence-redwood-nesw-snow,game:roughhewnfence-walnut-nesw-snow,game:roughhewnfencegate-aged-n-opened-free,game:roughhewnfencegate-birch-n-opened-free
1.7.2022 18:01:12.659 [VerboseDebug] Slow to load blocks game:pan-wooden,game:woodenpath-ns,game:woodenpath-we,game:pitkiln
1.7.2022 18:01:12.752 [VerboseDebug] Done blocks OnLoaded
1.7.2022 18:01:12.936 [VerboseDebug] Done items OnLoaded
```

**Comment by copygirl** [comment]
Some or all of this might have been fixed in #1981 / 1.17.0-pre.3.
Please reopen or make another issue if some keys still remain missing.
